products/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db    import models
+
+from users.models import Creator
+
+class Category(models.Model):
+    name        = models.CharField(max_length = 20)
+    created_at  = models.DateTimeField(auto_now_add = True)
+    modified_at = models.DateTimeField(auto_now = True)
+
+    class Meta:
+        db_table = 'categories'
+
+    def __str__(self):
+        return self.name
+
+class SubCategory(models.Model):
+    category    = models.ForeignKey(Category,on_delete = models.SET_NULL, null = True)
+    name        = models.CharField(max_length = 20)
+    created_at  = models.DateTimeField(auto_now_add = True)
+    modified_at = models.DateTimeField(auto_now = True)
+
+    class Meta:
+        db_table = 'sub_categories'
+
+    def __str__(self):
+        return self.name
+
+class Level(models.Model):
+    name        = models.CharField(max_length = 30)
+    created_at  = models.DateTimeField(auto_now_add = True)
+    modified_at = models.DateTimeField(auto_now = True)
+
+    class Meta:
+        db_table = 'levels'
+
+    def __str__(self):
+        return self.name
+
+class ProductStatus(models.Model):
+    name        = models.CharField(max_length = 30)
+    created_at  = models.DateTimeField(auto_now_add = True)
+    modified_at = models.DateTimeField(auto_now = True)
+
+    class Meta:
+        db_table = 'product_statuses'
```

Comment:
```suggestion
        db_table = 'product_status'
```
`status`자체가 보다 자연스러운 복수형 명사입니다.

Reply:
수정했습니다.